wandb/sdk/wandb_run.py

```diff
@@ -1669,9 +1669,28 @@ def log_artifact(self, artifact_or_path, name=None, type=None, aliases=None):
         if isinstance(aliases, str):
             aliases = [aliases]
         artifact.finalize()
+        self._assert_can_log_artifact(artifact)
```

Comment:
Does this break offline mode? It should be possible to run artifact logging runs without being connected to the internet: https://docs.wandb.com/library/technical-faq#can-i-run-wandb-offline

Reply:
Per Anni's comment below, I added a new condition to check for this in offline setting in the next commit